source/features/add-delete-to-pr-files.js

```diff
@@ -0,0 +1,21 @@
+import select from 'select-dom';
+import {h} from 'dom-chef';
+import * as pageDetect from '../libs/page-detect';
+
+const repoURL = pageDetect.getRepoURL();
+
+export default function () {
+	const findBranch = select('.head-ref').innerText;
+	const branchName = findBranch.includes(':') ? findBranch.split(':')[1] : findBranch;
+
+	const filesInfo = select.all('.file-info');
+	const generateURL = filesInfo.map(file => 'https://github.com/' + repoURL + '/delete/' + branchName + '/' + file.children[1].title);
```

Comment:
This will not work with GHE. Please apply the logic we use other places to account for different base URLs.